doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -211,6 +211,17 @@ Memory or Huge Pages are not considered by the Topology Manager. This can be don
 
 [Device Plugin Framework](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) advertises device hardware resources to kubelet with which vendors can implement plugins for devices that may require vendor-specific activation and life cycle management, and securely maps these devices to containers.
 
+Figure 3-2 shows in four steps how device plugins operate on a Kubernetes node:
+* 1: During setup, the cluster administrator (more in [3.2.5.1 Operator Pattern](chapter03.md#3251-operator-pattern)) knows or discovers (as per [3.2.1.5 Node Feature Discovery](chapter03.md#3215-node-feature-discovery)) what kind of devices are present on the different nodes, selects which devices to enable and deploys the associated device plugins.
+* 2: The plugin reports the devices it found on the node to the Kubelet device manager and starts its gRPC server to monitor the devices.
+* 3: A user submits a pod specification (workload manifest file) requesting a certain type of device.
+* 4: Kubelet decides which node to schedule workload on and which device on the node to assign to the pod's containers.
```

Comment:
nit-pick: the scheduler decides the node and kubelet handles the device allocation.
```suggestion
* 4: The scheduler determines a suitable node based on device availability and the local kubelet assigns a specific device to the pod's containers.
```

Reply:
> nit-pick: the scheduler decides the node and kubelet handles the device allocation.

@gkunz : Thank you!